examples/lang2/erc721/Cargo.toml

```diff
@@ -0,0 +1,58 @@
+[package]
+name = "erc721"
+version = "0.1.0"
+authors = ["XatMassacrE"]
```

Comment:
I am not sure if we are going to accept example authors to be different from the default. Have to ask ...